src/ui/ImageScaleView.js

```diff
@@ -354,7 +354,7 @@ exports = Class(ui.View, function (supr) {
         iw = bounds.width + bounds.marginLeft + bounds.marginRight;
         ih = bounds.height + bounds.marginTop + bounds.marginBottom;
       }
-    } else {
+    } else if (img){
```

Comment:
These changes are for handling non sprite images. we are not using non sprited images in ImageScaleView. But when we are setting image as null somewhere. So this checking is needed